packaging/vagrant/Vagrantfile

```diff
@@ -191,6 +216,21 @@ EOF2
     ch-run --version
   EOF
 
+  # Install ch-grow dependencies
+  c.vm.provision "ch-grow", type: "shell", privileged: true,
+                 inline: <<-EOF
+    yum -y install golang \
+                   go-md2man \
+                   python3.6 \
+                   python36-pip \
+                   skopeo
+
+    go get github.com/openSUSE/umoci
+    cd "/root/go/src/github.com/openSUSE/umoci" || exit 1
```

Comment:
Quotes not needed here.

Idiomatic `|| exit 1` is `set -e` at the beginning of the block, I think. That will cover the rest of the script too.